_topics/en/pcac.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Payments Contingent on Anything Computable (PCAC)
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: pcac
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BitVM
```

Comment:
MATT now could support this as well. Do we list all of them here as aliases? Or MATT is in the see also

Reply:
We can't list any existing topic in an alias as the list of topics on https://bitcoinops.org/en/topics/ is built from names + aliases, so having a topic and an alias with the same name would be a conflict.  I'm not actually sure what happens if we accidentally did that; I should make sure that fails or is otherwise tested for.  More directly, I think of MATT now as offering a superset of functionality to the behavior described on this page, kinda like how vaults could use CTV but CTV can also do other things.  We wouldn't list CTV as an alias for vaults, so I don't think MATT should be an alias for ACC.

ZKCP and BitVM are strictly for ACCs and I don't plan to split them off into separate topics unless one of them starts getting mentioned a lot.